src/data/markdown/docs/30 k6-browser/02 API/02 BrowserContext/setDefaultTimeout.md

```diff
@@ -21,13 +21,16 @@ Sets the default maximum timeout for all methods accepting a `timeout` option in
 
 <CodeGroup labels={[]}>
 
-<!-- eslint-skip -->
-
 ```javascript
-const context = browser.newContext();
-context.setDefaultTimeout(1000); // 1s
-const page = context.newPage();
-page.click('h2'); // times out
+import { chromium } from 'k6/experimental/browser';
+
+export default function () {
+  const browser = chromium.launch();
+  const context = browser.newContext();
+  context.setDefaultTimeout(1000); // 1s
+  const page = context.newPage();
+  page.click('h2'); // times out
+}
```

Comment:
Since `click` is async:

```suggestion
export default async function () {
  const browser = chromium.launch();
  const context = browser.newContext();
  context.setDefaultTimeout(1000); // 1s
  const page = context.newPage();
  await page.click('h2'); // times out
}
```